### DISCUSSION TITLE
ScriptReference/Resources

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#245] [Notes]
# ScriptReference/Resources

https://docs.unity3d.com/ScriptReference/Resources.html

**Comment by vertxxyz** [comment]
**[Special folder names - Resources](https://docs.unity3d.com/Manual/SpecialFolders.html#:~:text=are%20in%20subfolders.-,Resources,-You%20can%20load)**
> If the Resources folder is an Editor subfolder, the Assets in it are loadable from Editor scripts, but are removed from builds.
